scripts/hubot-netrunnerdb-cards.coffee

```diff
@@ -239,6 +239,36 @@ compareCards = (card1, card2) ->
 	else
 		return 0
 
+cardMatches = (card, cond) ->
+	switch cond.op
```

Comment:
It's a good question. In theory no, because invalid conditions have already been stripped out. I guess it should default to some truthy value just in case though (i.e. it will ignore invalid conditions). I'll make the tweak and update.